.woke.yaml

```diff
@@ -2,9 +2,10 @@ ignore_files:
   # Ignore ingress charm library as it uses non compliant terminology:
   # whitelist.
   - lib/charms/nginx_ingress_integrator/v0/ingress.py
+  - lib/charms/nginx_ingress_integrator/v0/nginx_route.py
 rules:
   # Ignore "master" - the database relation event received from the library
   # and uwsgi use this terminology.
   - name: master
   # Ignore "blacklist" -Indico uses this term
-  - name: blacklist
\ No newline at end of file
+  - name: blacklist
```

Comment:
Is this still required for this repository?

Reply:
No, removed.